maps/randomvaults/objects.dm

```diff
@@ -136,3 +139,20 @@
 /obj/item/weapon/paper/iou
 	name = "paper- 'IOU'"
 	info = "I owe you a rod of destruction. Redeemable at Milliway's at the end of time."
+
+/obj/machinery/floodlight/on
+
+	New()
```

Comment:
So wait, what did this do?

This isn't the infinite feedback loop thing that I've heard about, is it?